lms/djangoapps/course_home_api/dates/v1/tests/test_views.py

```diff
@@ -0,0 +1,54 @@
+"""
+Tests for Dates Tab API in the Course Home API
+"""
+
+
+import ddt
+
+from django.urls import reverse
+
+from course_modes.models import CourseMode
+from lms.djangoapps.course_home_api.tests.utils import BaseCourseHomeTests
+from student.models import CourseEnrollment
+
+
+@ddt.ddt
+class DatesTabTestViews(BaseCourseHomeTests):
```

Comment:
Is it worth adding a query count test? Like, loading blocks from mongo aside, it would help make sure that we are prefetching correctly and all that. In a fantasy dream world, this view could just be 1 query if everything were prefetched, yeah?

Reply:
Upon doing a quick test for this, we determined a query count test would not have significant help right now.